src/components/search/curated/css/index.tsx

```diff
@@ -1,17 +1,49 @@
-import React from 'react'
-import cssPageData from './css-page-data'
+import * as React from 'react'
+import Card from 'components/pages/home/card'
 import SearchCuratedEssential from '../curated-essential'
+import LoadDataFromSanity from './sanity-loading-data'
+import {slice} from 'lodash'
+import Collection from 'components/pages/home/collection'
 
 const SearchCSS = () => {
+  const data = LoadDataFromSanity()
+  const {resources} = data || {}
+  const levels = slice(resources, 0, 3)
+  const rest = slice(resources, 3, resources?.length)
```

Comment:
I wasn't how to pin point collections I want without hardcoding their `slug` or `id`, so I ended up using their order in Sanity.

<img src="https://user-images.githubusercontent.com/25487857/107398692-2b909600-6b00-11eb-893d-cfc7da584d5a.png" width="400">

Reply:
beginner, intermediate, advanced are "template collections" and should probably be named/slugged specifically and used if present. Alternatively the resource could be slugged "levels" and contain these three as a group in that resource (nested)

This slice approach is worse than hard coding imo

Reply:
as a preference I like `take` and `drop` over `slice` for this situation https://lodash.com/docs/4.17.15#take

```js
levels = take(resources, 3)
rest = drop(resources, 3)
```

just easier on the eyes imo